themes/jaeger-docs/layouts/index.redirects

```diff
@@ -1,5 +1,5 @@
-{{- $versions     := sort (readDir "content/docs") "Name" "desc" }}
-{{- $latest       := (index ($versions) 0).Name }}
+{{- $versions     := .Site.Params.versions }}
+{{- $latest       := .Site.Params.latest }}
 {{- $originalDocs := readDir "content/docs/1.6" -}}
```

Comment:
is this correct that it's reading 1.6 explicitly?

Reply:
Yes, because that's the only version for which there were URLs of the form `/docs/getting-started` (i.e. without a version number). So it's reading that directory to figure out which URLs with that structure that it needs to route to, for example, `docs/latest/getting-started`.

Another option would be to just hard-code those and leave them as-is. That's probably a more future-proof approach.